programming/leds.md

```diff
@@ -6,37 +6,34 @@ title: Brain Board LED API
 Brain Board LED API
 ===================
 
-<div class="info">
-This documentation refers to a feature which is only available on the physical robot kits.
-</div>
+The KCH board provides diagnostic information about the state of your robot through a collection of preconfigured and user-controllable LEDs.
```

Comment:
I think when we built it, we just called it "KCH", with the vague idea the "H" meant "Hat". Calling it "KCH Board" feels a little like "ATM Machine". Personally, I much prefer just referring to it as "the KCH".

Reply:
We had this discussion a while back. 
Gramatically is it fine:
(Kajetan Champlewski)(Hardware on Top) Board
We thought the word board next to it gives context to the reader that the KCH, if they didnt know what it was, was one of the boards.